questionnaires/chair_rises/chair_rises_armt.json

```diff
@@ -0,0 +1,403 @@
+[
+    {
+        "field_name": "chair_rises_instructions",
+        "form_name": "chair_rises",
+        "section_header": "",
+        "field_type": "info",
+        "field_label": "Chair Rises Instructions",
+        "select_choices_or_calculations": [
+            {
+                "code": "Part1",
+                "label": "Comfortable clothing and appropriate footwear should be worn for this test."
+            },
+            {
+                "code": "Part2",
+                "label": "Sit on a straight back chair (for additional stability, place the rear legs of the chair against a wall so that it cannot slide backwards)."
+            },
+            {
+                "code": "Part3",
+                "label": "Press button A (activity button A is located at the top right-hand side of the watch face)"
+            },
+            {
+                "code": "Part4",
+                "label": "Note! If this is the first time you have started an activity, select the checkbox next to 'Walk' and 'Cardio' to add these activities to your favorites."
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "info-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "chair_rises_instructions_2",
+        "form_name": "chair_rises",
+        "section_header": "",
+        "field_type": "info",
+        "field_label": "Chair Rises Instructions",
+        "select_choices_or_calculations": [
+            {
+                "code": "Part1",
+                "label": "Using the watch face touchscreen (C), select 'Cardio'"
+            },
+            {
+                "code": "Part2",
+                "label": "'Ready' will appear on the watch face'."
+            },
+            {
+                "code": "Part3",
+                "label": "Press button A to start the activity timer'"
+            },
+            {
+                "code": "Part4",
+                "label": "Go! Stand up and then sit down as quickly as you can 10 times'"
+            },
+            {
+                "code": "Part5",
+                "label": "When you have completed 10 chair rises, Stop!"
+            },
+            {
+                "code": "Part6",
+                "label": "Check the clock timer on the watch face and sit still for a 1-minute recovery period. After 1 minute of recovery, press button A to stop the activity timer'."
+            },
+            {
+                "code": "Part7",
+                "label": "Save the activity by pressing the green tick on the watch face touchscreen (C)."
```

Comment:
I thought there was an image (of garmin) too in the app, maybe using descriptive text input would be better here?